### ISSUE TITLE
Workspace name as a prefix of helm instance name

### ISSUE THREAD

**Issue by gauthiermartin** [astronomer/astronomer#336]
**Feature request**
It would be nice to have the workspace name replicated into the helm installation name. It would greatly help instance support and debug.

**Describe the solution**
It would be great to have the name of the instance used while creating a new instance in orbit ui to prefix the randomly generated name in helm

**Additional context**
Here is an image of the workspace name field 
![Imgur](https://i.imgur.com/nQWDpMp.png)

**Comment by paolaperaza** [comment]
@schnie I'm cleaning out the last of the issues in this repo - this relevant to transfer over and/or merge with any other existing issues that you know of?

(cc: @gauthiermartin we're privatizing this GitHub repo whilst working on ways to still allow public trackable feature requests. Appreciate you posting in here back in June - not letting it get lost in the mix 😊)

**Comment by schnie** [comment]
Yes still seems relevant. In the past, we weren't showing workspace ids, but we've made some recent changes so the CLI will show the id, and so will the UI. This id should be passed through to all objects that helm creates as a label called `workspace`.

Maybe not perfect, but should at least help out here a little bit.
